### DISCUSSION TITLE
File 'tsconfig/nextjs.json' not found.ts

### DISCUSSION THREAD

**Discussion by BuchananDevOps** [vercel/turborepo#1636] [Help] [answered]
Just trying to test turbo repo for an upcoming project and after running the npx create-turbo, selecting pnpm or yarn I get the same errors. You'd think the examples would be plug and play...

File 'tsconfig/nextjs.json' not found.ts
File 'tsconfig/react-library.json' not found.ts

**Comment by chapmanio** [comment]
Did you have a resolution for this?  I suspect it's something lingering elsewhere on my machine with `yarn` or `pnpm` but I'm facing the same issue.

**Reply by chapmanio** [reply]
It's worth noting this only happens when I'm using `yarn v3.2.3` (via `corepack enable`) if I switch back to yarn classic it will play nice.

**Comment by anthonyshew** [comment, marked as answer]
We've updated our examples recently! If you're still seeing issues, please open up an issue. 🙏
